sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/MethodShouldBeNameAccordingToSynchronicityTest.cs

```diff
@@ -28,16 +28,55 @@ namespace SonarAnalyzer.UnitTest.Rules
     public class MethodShouldBeNameAccordingToSynchronicityTest
     {
         [TestMethod]
+        [DataRow("4.0.0")]

```

Comment:
Did you try if we could use shorter version numbers? This is minor, but I guess it could be convenient to specify just `4.0` or `4.0.*` and nuget to automatically download the latest patch, e.g. `4.0.15`.

Reply:
I only took the older version available on NuGet for each package.

Reply:
A bug fix shouldn't change anything on our side. What we care is to say this works on old and works on latest.